index.js

```diff
@@ -416,6 +424,13 @@ converters.ObjectExpression = (path, context) => {
   };
 };
 
+converters.FunctionDeclaration = (path, context) => {
+  return {
+    kind: "FunctionDeclaration",
+    id: convert(path.get("id"), context),
+  };
+}
```

Comment:
Can you abstract out the logic for converting the other function types and get the same interface as this:

https://github.com/atlassian/extract-react-types/blob/b644deb4558e166c9d9b8c7833e71eecd83a2ac3/index.js#L281-L315